src/Resources/contao/dca/tl_templates.php

```diff
@@ -227,7 +227,7 @@ public function addNewTemplate()
 		foreach ($kernel->getContainer()->get('contao.resource_provider')->getResourcesPaths() as $strModule => $strFolder)
 		{
 			// Continue if there is no templates folder
-			if ($strModule == 'repository' || !is_dir($strFolder))
```

Comment:
Can I still ask you to fix it here? It's kind of confusing to merge faulty code.